### ISSUE TITLE
Rename of module does not handle import aliases correctly

### ISSUE THREAD

**Issue by davidruffner** [python-rope/rope#229]
I am using rope to rename modules in my project and came across a problem with how it handles aliases. Here is a minimal example that shows this behavior. In this case, `mod2.py` imports `mod1.py` and so should import `mod1.py` with a new name when `mod1.py`'s name is changed. However, in this case no changes are actually made in `mod2.py`. I am using version 0.10.7.

Here is my example:
```python
from rope.refactor.rename import Rename
from rope.base.project import Project

project = Project('.')

mod1 = project.root.create_file('mod1.py')
mod1.write(
    'class Test(object):\n'
    '    def __init__(self):'
    '        print "hello world"')

mod2 = project.root.create_file('mod2.py')
mod2string = 'import mod1 as m\n' + 'm.Test()'
mod2.write(mod2string)

new_name = 'AAAAAAAA'
changes = Rename(project, mod1).get_changes(new_name)
project.do(changes)

expected = 'import ' + new_name + ' as m\n' + 'm.Test()'
with open('mod2.py', 'r') as f:
    actual = f.read()
print '~' * 50
print actual
print '~' * 50
print expected
assert  actual == expected

```
I get the following output
```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import mod1 as m
m.Test()
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import AAAAAAAA as m
m.Test()

---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-52-b92f47d557f0> in <module>()
     25     print '~' * 50
     26     print expected
---> 27     assert  actual == expected
     28 

AssertionError: 
```

**Comment by soupytwist** [comment]
Hmm, I can reproduce it. Works as expected without the alias. Thanks for reporting, I'll take a closer look.

**Comment by davidruffner** [comment]
Thanks for your help! Your fix (#230) is working for me.

**Comment by soupytwist** [comment]
Thanks for checking!
